README.md

```diff
@@ -83,13 +89,13 @@ If you always used Copier with this project, getting last updates with Copier is
 
 ```bash
 cd ~/path/to/your/downstream/scaffolding
-copier update
+copier update --UNSAFE
 ```
 
 Copier will ask you all questions again, but default values will be those you answered
 last time. Just hit <kbd>Enter</kbd> to accept those defaults, or change them if
-needed... or you can use `copier --force update` instead to avoid answering again all
-things.
+needed... or you can use `copier --force update --UNSAFE` instead to avoid answering
```

Comment:
In copier 8+, all flags are *after* the subcommand. This has changed for ergonomy, and because there's a new subcommand that required a different set of flags. 

Thus, this has changed too:

```suggestion
needed... or you can use `copier update --UNSAFE --force` instead to avoid answering
```